torchvision/utils.py

```diff
@@ -259,7 +261,14 @@ def draw_bounding_boxes(
             draw.rectangle(bbox, width=width, outline=color)
 
         if label is not None:
-            margin = width + 1
+            box_margin = 1
+            margin = width + box_margin
+            if fill_labels:
+                left, top, right, bottom = draw.textbbox((bbox[0] + margin, bbox[1] + margin), label, font=txt_font)
+                draw.rectangle(
+                    (left - box_margin, top - box_margin, right + box_margin, bottom + box_margin), fill=color
+                )
+                label_color = "black"
```

Comment:
Do we have any reason to force `label_color` to black? Could we just make `black` the default, but let the user specify it if they wanted to?